src/main/java/com/example/SS2_Backend/controller/HomeController.java

```diff
@@ -38,6 +42,15 @@ public CompletableFuture<ResponseEntity<Response>> solveStableMatching(@RequestB
         return CompletableFuture.completedFuture(stableMatchingSolver.solveStableMatching(object));
     }
 
+    /*
+    * Đây là phần chạy RBO (Request Body Optimization) để giải Stable Matching Problem
+    * */
+    @Async("taskExecutor")
+    @PostMapping("/stable-matching-rbo-solver")
+    public CompletableFuture<ResponseEntity<Response>> solveStableMatching(@RequestBody NewStableMatchingProblemDTO object) {
+        return CompletableFuture.completedFuture(stableMatchingSolverRBO.solveStableMatching(object));
+    }
+
```

Comment:
the route should be named differently, `/stable-matching-solver-v1.1` for example

Reply:
-> Không v đề gì, nếu sau này code tối ưu chạy ổn thì sẽ đổi lại thành /stable-matching-solver như cũ, xóa cái old đi